docs/user-guide/global-configurations/lock-deployment-config.md

```diff
@@ -0,0 +1,103 @@
+# Lock Deployment Config
+
+## Introduction [![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
+
+The [Deployment Template](../../reference/glossary.md#base-deployment-template) might contain certain configurations intended for the DevOps team (e.g., `ingress`), and not meant for developers to modify. 
+
+Therefore, Devtron allows super-admins to restrict such fields from modification or deletion.
+
+![Figure 1: Preventing Changes to Locked Keys](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/lock-dt/not-eligible-changes.jpg)
+
+This stands true for deployment templates in:
+* [Base configuration](../../user-guide/creating-application/deployment-template.md)
+* [Environment-level configuration](../../user-guide/creating-application/environment-overrides.md)
+
+**How is this different from the [Protect Configuration](../../user-guide/creating-application/config-approval.md) feature?**
+
+The 'protect configuration' feature is meant to verify the edits by introducing an approval flow for any changes made to the configuration files, i.e., Deployment template, ConfigMaps, and Secrets. This is performed at application-level.
+
+Whereas, the 'lock deployment config' feature goes one step further. It is meant to prevent any edits to specific keys by non-super-admins. This applies only to deployment templates and is performed at global-level.
+
+---
+
+## Locking Deployment Keys
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to lock deployment keys.
+{% endhint %}
+
+1. Go to **Global Configurations** → **Lock Deployment Config**. Click **Configure Lock**.
+
+    ![Figure 2: Configure Lock Button](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/lock-dt/lock-deployment-config.jpg)
+
+2. (Optional) Click **Refer Values.YAML** to check which keys you wish to lock.
+
+    ![Figure 3: Values.YAML File](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/lock-dt/values-yaml.jpg)
+
+3. Enter the keys inside the editor on the left-hand side, e.g., `resource.limits`.
+
+    ![Figure 4: Referring Values.YAML File](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/lock-dt/side-by-side.jpg)
```

Comment:
Suggestion: Can we remove this screenshot, the empty state does not seem to add much value. Also, the two sentences together would be better. (something like below)

Enter the keys inside the editor on the left-hand side, e.g., `autoscaling.MaxReplicas`. Use <a href="https://goessner.net/articles/JsonPath/index.html" target="_blank">JSONpath expressions</a> to enter specific keys, lists, or objects to lock.